manifest_staging/charts/aad-pod-identity/templates/podidentityexceptions.yaml

```
@@ -0,0 +1,16 @@
+{{- if not (empty .Values.podIdentityExceptions) -}}
+{{- range $key, $value := .Values.podIdentityExceptions }}
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzurePodIdentityException
+metadata:
+  name: {{ default $key $value.name }}
+  namespace: {{ default $.Release.Namespace $value.namespace }}
+  labels:
+    {{- include "aad-pod-identity.labels" $ | nindent 8 }}
+spec:
+  podLabels:    
+        {{- range $key2, $value2 := $value.podLabels }}
+        {{ $key2 }}: {{ $value2 }}
+        {{- end }}
```

we can use `toYaml` to convert `podLabels` map to yaml directly instead of using `range`:

```suggestion
  podLabels: {{ toYaml $value.podLabels | nindent 4 }}
```

---

@ams0 could you update the PR to use `{{ toYaml $value.podLabels | nindent 4 }}`?